indexer/src/test/java/au/org/aodn/esindexer/service/VocabServiceIT.java

```diff
@@ -115,6 +128,29 @@ void testProcessParameterVocabs() throws IOException, JSONException {
         );
     }
 
+    @Test
```

Comment:
Add comment on what the test case is about and why we need the logic

Add test case to show that vocabs have value, when there is a invalid load, the vocabs will not be discarded